core/app/models/workarea/search/storefront/product.rb

```diff
@@ -59,7 +59,8 @@ def content
             category_names: category_names,
             description: catalog_content,
             details: details_content,
-            facets: facets_content
+            facets: facets_content,
+            tags: model.tag_list
```

Comment:
I think this should be in `keywords` or maybe `facets` - `content` gets fulltext-analyzed so we def wouldn't tags to get that analysis for this purpose.